### ISSUE TITLE
How to solve "error trying to exec 'f951': execvp: No such file or directory"

### ISSUE THREAD

**Issue by sepastian** [snavely/bundler_sfm#58]
Hello, thanks for bundler, just trying it out.

Just a little note for anyone who might run into this. I'm compiling on 

```
$ cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux buster/sid"
NAME="Debian GNU/Linux"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

When running `make` according to your instructions, I ran into `error trying to exec 'f951': execvp: No such file or directory`. To overcome this, I had to `sudo apt install gfortran`. Afterwards, `make` would execute without errors.

**Comment by snavely** [comment]
You mentioned that you tried

> sudo apt install gfortran

but isn't the correct command

> sudo apt-get install gfortran

? In theory if you have the gfortran package installed it should be able to
find the fortran compiler...

If you are still having problems, I found this page which has a hacky fix:
https://stackoverflow.com/questions/13293762/gfortran-error-trying-to-exec-f951-execvp-no-such-file-or-directory

Noah

**Comment by sepastian** [comment]
As mentioned, what I wrote is intended just as a note to other users running into the same issue.

Concerning `apt`: the `apt` command can do what `apt-get` does, but it's more recent and more comfortable to use. See [this article](https://itsfoss.com/apt-vs-apt-get-difference/), for details. So `sudo apt install gfortran` does work.

As I wrote, I don't have any problems, but thanks for your reply.
